README.md

```diff
@@ -283,6 +135,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your license in your readme file is no longer active. Please correct it. It is critical to include a license in your project because it is a document that provides legally binding guidelines for use and imposes restrictions on how your project can be used. 📗 🙂